tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+    'ouroboros/tkinter/messagebox.py',
+    'ouroboros/tkinter/simpledialog.py',
+    'ouroboros/tkinter/tix.py',
+    'ouroboros/token.py',
+    'ouroboros/tokenize.py',
+    'ouroboros/tracemalloc.py',
+    'ouroboros/tty.py',
+    'ouroboros/turtledemo/bytedesign.py',
+    'ouroboros/turtledemo/chaos.py',
+    'ouroboros/turtledemo/clock.py',
+    'ouroboros/turtledemo/colormixer.py',
+    'ouroboros/turtledemo/forest.py',
+    'ouroboros/turtledemo/fractalcurves.py',
+    'ouroboros/turtledemo/lindenmayer.py',
+    'ouroboros/turtledemo/minimal_hanoi.py',
+    'ouroboros/turtledemo/nim.py',
+    'ouroboros/turtledemo/paint.py',
+    'ouroboros/turtledemo/peace.py',
+    'ouroboros/turtledemo/penrose.py',
+    'ouroboros/turtledemo/planet_and_moon.py',
+    'ouroboros/turtledemo/round_dance.py',
+    'ouroboros/turtledemo/tree.py',
+    'ouroboros/turtledemo/two_canvases.py',
+    'ouroboros/turtledemo/wikipedia.py',
+    'ouroboros/turtledemo/yinyang.py',
+    'ouroboros/types.py',
+    'ouroboros/unittest/__main__.py',
+    'ouroboros/unittest/main.py',
+    'ouroboros/unittest/result.py',
+    'ouroboros/unittest/runner.py',
+    'ouroboros/unittest/signals.py',
+    'ouroboros/unittest/suite.py',
+    'ouroboros/unittest/test/__main__.py',
+    'ouroboros/unittest/test/_test_warnings.py',
+    'ouroboros/unittest/test/dummy.py',
+    'ouroboros/unittest/test/support.py',
+    'ouroboros/unittest/util.py',
+    'ouroboros/urllib/error.py',
+    'ouroboros/urllib/response.py',
+    'ouroboros/venv/__main__.py',
+    'ouroboros/warnings.py',
+    'ouroboros/wave.py',
+    'ouroboros/wsgiref/headers.py',
+    'ouroboros/wsgiref/simple_server.py',
+    'ouroboros/wsgiref/util.py',
+    'ouroboros/wsgiref/validate.py',
+    'ouroboros/xml/dom/NodeFilter.py',
+    'ouroboros/xml/dom/domreg.py',
+    'ouroboros/xml/dom/expatbuilder.py',
+    'ouroboros/xml/dom/pulldom.py',
+    'ouroboros/xml/etree/ElementInclude.py',
+    'ouroboros/xml/etree/cElementTree.py',
+    'ouroboros/xml/parsers/expat.py',
+    'ouroboros/xml/sax/_exceptions.py',
+    'ouroboros/xml/sax/handler.py',
+    'ouroboros/xml/sax/saxutils.py',
+    'ouroboros/xml/sax/xmlreader.py',
+]
+
+
+def ouroboros_folder():
+    """Return the folder where the ouroboros repo was cloned."""
+    os.chdir(HERE)
+    os.chdir('../')
+    return os.path.join(os.getcwd(), 'ouroboros', 'ouroboros')
+
+
+def clone_update_repo():
+    """Clone or update the ouroboros repository inside the voc folder."""
+    print('\nUPDATING OUROBOROS CODE')
+    print('=======================')
+    url = 'https://github.com/pybee/ouroboros.git'
+
+    repo_folder = ouroboros_folder()
+    if os.path.isdir(repo_folder):
+        print('Pulling from master on repo: {}\n'.format(url))
+        args = ['git', 'pull', 'origin', 'master']
+    else:
+        print('Cloning repo: {}\n'.format(url))
+        args = ['git', 'clone', url]
+
+    p = Popen(args, stdout=PIPE, stderr=PIPE, cwd=repo_folder)
```

Comment:
It is failing for me, saying that the `ouroboros/ouroboros` directory doesn't exist:

```
$ python tests/compile_ouroboros.py 

UPDATING OUROBOROS CODE
=======================
Cloning repo: https://github.com/pybee/ouroboros.git

Traceback (most recent call last):
  File "tests/compile_ouroboros.py", line 658, in <module>
    clone_update_repo()
  File "tests/compile_ouroboros.py", line 551, in clone_update_repo
    p = Popen(args, stdout=PIPE, stderr=PIPE, cwd=repo_folder)
  File "/home/elias/.pythonz/pythons/CPython-3.4.4/lib/python3.4/subprocess.py", line 859, in __init__
    restore_signals, start_new_session)
  File "/home/elias/.pythonz/pythons/CPython-3.4.4/lib/python3.4/subprocess.py", line 1463, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: '/home/elias/src/voc-dev/voc/ouroboros/ouroboros'
```